docs/Options.md

```diff
@@ -162,7 +162,7 @@ By default, ExegesisRunner will turn `exegesis.HttpError`s (such as errors
 generated from `context.makeError()`, `exegesis.ValidationError`s, or any error
 with a `.status` into JSON replies with appropriate error messages.  If you want
 to handle these errors yourself, set this value to false, alternatively set the value
-to a `function(err)` function that handles the error returning a `HttpResult` object.
+to a `function(err, req)` function that handles the error returning a `HttpResult` object.
```

Comment:
One small suggestion; maybe make this a `function(err, { req })` - this way if we want to add something else to this function signature later, it won't get out of hand.  :)